src/styles.ts

```diff
@@ -31,6 +31,36 @@ if (uiMode === 'dark') {
 
 export const colors = uiColors
 
+export const darkTheme = StyleSheet.create({
+  view: {
+    backgroundColor: PV.Colors.black
+  },
+  text: {
+    color: PV.Colors.white
+  },
+  tabbar: {
+    backgroundColor: PV.Colors.black
+  },
+  tabbarItem: {
+    tintColor: PV.Colors.blue
+  }
+})
+
```

Comment:
Something that I haven’t wrapped my brain around with darkMode…those vars work for the StyleSheets, but what about managing non-StyleSheet colors like in say the navigationOptions or tab styles?

Reply:
I know... this was a limitation unfortunately fir the inactive state of the tabbarItems. I was able to adjust the active state color using hooks but not the inactive state because that is set on initial setup. I thought maybe we will have to decide upon a specific color for the states of tabbarItems. I left it as gray because it seemed the best choice for both themes

Reply:
Alright, I'm cool with just using a single style for these types of elements until we figure it out.